assets.tf

```diff
@@ -3,13 +3,13 @@ resource "template_dir" "bootstrap-manifests" {
   source_dir      = "${replace(path.module, path.cwd, ".")}/resources/bootstrap-manifests"
   destination_dir = "${var.asset_dir}/bootstrap-manifests"
 
-  vars {
-    hyperkube_image   = "${var.container_images["hyperkube"]}"
-    etcd_servers      = "${join(",", formatlist("https://%s:2379", var.etcd_servers))}"
-    cloud_provider    = "${var.cloud_provider}"
-    pod_cidr          = "${var.pod_cidr}"
-    service_cidr      = "${var.service_cidr}"
-    trusted_certs_dir = "${var.trusted_certs_dir}"
+  vars = {
+    hyperkube_image   = var.container_images["hyperkube"]
```

Comment:
What about 0.11 compatibility? If we don't remove interpolation, we should be able to achieve it. Or should we not focus on that?

That should also make upgrades smoother.

Reply:
Scratch that: https://github.com/kinvolk/PROJECT-Lokomotive-Kubernetes/issues/231#issuecomment-570206965.

Reply:
Unfortunately it is not possible to keep compatibility with Terraform 0.11 in this module.
In Terraform 0.11 it was possible to flatten a list by using this syntax: https://github.com/kinvolk/terraform-render-bootkube/blob/555822f998208058bd60baf67138289ba3cd2ccb/tls-k8s.tf#L53-L60.

In Terraform 0.12 it'll create a list of lists (www.terraform.io/upgrade-guides/0-12.html#referring-to-list-variables) and will cause a type assertion error, in order to create a flat list the `flatten` function is used in 0.12, but it is not available in 0.11.